Extensions/ServiceNow/Src/Tasks/CreateAndQueryChangeRequest/CreateAndQueryChangeRequestV2/task.json

```diff
@@ -375,7 +375,7 @@
+            "Body": "{  \"u_correlation_id\": \"{{#newGuid}}{{/newGuid}}\",\"u_type\": \"$(changeType)\",\"u_x_mioms_azpi_eline_metadata\": \"Collection URI: $(system.collectionUri) \\r\\n Project: $(system.teamProject) \\r\\n StageName: $(system.stageName) \\r\\n RequestedFor: $(build.requestedFor)\", \"u_short_description\": \"$(shortdescription)\"{{#equals changeType 'Normal' 1}}{{#if description}}, \"u_description\": \"$(description)\"{{/if}}{{#if category}}, \"u_category\": \"$(category)\"{{/if}}{{#if priority}}, \"u_priority\": \"$(priority)\"{{/if}}{{#if risk}}, \"u_risk\": \"$(risk)\"{{/if}}{{#if impact}}, \"u_impact\": \"$(impact)\"{{/if}}{{#if configurationitem}}, \"u_cmdb_ci\": \"$(configurationitem)\" {{/if}}{{#if assignmentgroup}}, \"u_assignment_group\": \"$(assignmentgroup)\"{{/if}}{{/equals}}, \"u_requested_by_date\": \"{{toDateTimeFormat '$(system.checksStartTime)' 'dd-MM-yyyy HH:mm:ss UTC'}}\", \"u_start_date\": \"{{#if schedulestarttime}}{{toDateTimeFormat '$(schedulestarttime)' 'dd-MM-yyyy HH:mm:ss UTC'}}{{/if}}\",  \"u_end_date\": \"{{#if scheduleendtime}}{{toDateTimeFormat '$(scheduleendtime)' 'dd-MM-yyyy HH:mm:ss UTC'}}{{/if}}\" {{#if otherParameters}}{{toCommaSeparatedKeyValueList otherParameters true}}{{/if}}{{#equals changeType 'Standard' 1}}{{#if standardChangeTemplate}} , \"template_id\": \"$(standardChangeTemplate)\" {{/if}}{{/equals}}, \"u_x_mioms_azpipeline_stage_id\": \"$(system.stageId)\", \"u_x_mioms_azpipeline_build_id\": \"$(build.buildId)\", \"u_x_mioms_azpi__stage_attempt\": \"$(checks.stageAttempt)\" }",
```

Comment:
Why are we using get call with "x_mioms_azpipeline_stage_attempt" but POST call contains "u_x_mioms_azpi__stage_attempt" ?

Reply:
So, if you observe we make a `GET` call from `change_requests` table while as when we post we post to `change_request_imports` table. Hence, the column name difference.

Now, how the `POST` to `change_request_imports` puts data into `change_requests` table is, we have a mapping done between these both. So, whenever we import the data is populated via the same. 

Mapping link: https://<Instance-name>.service-now.com/v_ws_editor.do?sysparm_query=name=x_mioms_azpipeline_change_request_import